### DISCUSSION TITLE
Right-clicking on desktop icons restarts Windows Explorer

### DISCUSSION THREAD

**Discussion by pcdogs** [valinet/ExplorerPatcher#3153] [Q&A] [answered]
I installed ExplorerPatcher and when it's enabled it crashes the explorer, I know it's the patcher because I had no issues on Win11 Explorer. How do i fix it?

**Comment by pyrates999** [comment, marked as answer]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.  These should be screen shots.  If you can, please change it to english first.

Please include any other software you have installed that modifies the UI of windows.

duplicate of: https://github.com/valinet/ExplorerPatcher/issues/2218

please close this

Update your windows as this was already fixed by Microsoft in the current release of windows.

**Reply by pcdogs** [reply]
I'll send it over later and try checking for updates, thank you :)

**Reply by pcdogs** [reply]
Thank you so much, it was just the OS not being updated breaking it
